src/hooks/useTasks/useTasks.tsx

```diff
@@ -67,13 +82,49 @@ export const useTasks: UseTasks = () => {
     }
   }, [syncTimestamps])
 
+  const mockData: TaskTypes[] = [
```

Comment:
NIT: Perhaps you can move it out of function: `export const useTasks: UseTasks = () => {`